README.md

```diff
@@ -61,6 +62,10 @@ GLOBAL_QUERY_STRINGS_PARAMS = {}
 #### GLOBAL_QUERY_STRINGS_PARAMS 
 `GLOBAL_QUERY_STRINGS_PARAMS` takes a dictionary of global query strings to set e.g.: `{"foo": "bar", "lorem": "ipsum"}`. The query strings found in `GLOBAL_QUERY_STRINGS_PARAMS` would be replaced / added to the urls found by the parser.
 
+#### GLOBAL_QUERY_STRINGS_EXCLUDE_PATHS_LIST
+`GLOBAL_QUERY_STRINGS_EXCLUDE_PATHS_LIST` is a list of paths to exclude that takes place in the middleware only. 
+All the paths in this list will bypass the middleware. e.g.: `["/admin"]`
```

Comment:
You could specify that it will exclude all the paths that start by the string will be excluded.

Reply:
Fixed by b29f05f